plugins/include/amxmisc.inc

```diff
@@ -891,8 +919,8 @@ stock precache_player_model(const name[], &id = 0)
 	formatex(model, charsmax(model), "models/player/%s/%sT.mdl", name, name);
 
 	if(file_exists(model))
-		id = precache_model(model);
+		id = precache_generic(model);
 
-	replace_string(model, charsmax(model), "T.mdl", ".mdl");
+	replace_string(model[strlen(model) - 4], charsmax(model), "T.mdl", ".mdl");
```

Comment:
Still not the best option, magic `4` should be replaced with something like `sizeof("T.mdl") - 1`. But anyway `replace_string` doesn't fit good for me.

Reply:
@WPMGPRoSToTeMa I changed it to `charsmax` because `sizeof` would output `6` instead.

What do you suggest using instead `replace_string`? I couldn't figure out a way to manually insert a character without using functions.

Reply:
@OciXCrom `copy` maybe.